README.md

```diff
@@ -14,13 +14,16 @@ The buildpack will do the following:
 * Executes `rustup` to install a Rust toolchain to a layer marked `build` and `cache` with installed commands on `$PATH`
+* If `$BP_RUST_TARGET` is set, executes `rustup target add` to install an additional Rust target.
+* If `$BP_RUST_TARGET` is not set and the build is running on the Paketo Tiny stack, then the Rust Linux musl target will be automatically added.
 
 ## Configuration
 | Environment Variable      | Description                                                                                                                                                                                                                                                                  |
 | ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `$BP_RUSTUP_ENABLED`      | Configure rustup to be enabled. This means that rustup will be used to install Rust. Default value is `true`. Set to false to use another Rust toolchain provider like [rust-dist](https://github.com/paketo-community/rust-dist).                                           |
 | `$BP_RUST_TOOLCHAIN`      | Rust toolchain to install. Default `stable`. Other common values: `beta`, `nightly` or a specific versin number. Any [acceptable value for a toolchain](https://dev-doc.rust-lang.org/beta/edition-guide/rust-2018/rustup-for-managing-rust-versions.html) can be used here. |
 | `$BP_RUST_PROFILE`        | Rust profile to install. Default `minimum`. Other acceptable values: `default`, `complete`. See [Rustup docs for profile](https://rust-lang.github.io/rustup/concepts/profiles.html).                                                                                        |
+| `$BP_RUST_TARGET`         | Additional Rust target to install. Default ``, so nothing additional is installed. If there is no user-specified target and the build is running on the Paketo Tiny stack, then the Linux musl target is automatically added. Run `rustup target list` to see what valid targets exist.                                                                                                                                 |
 | `$BP_RUSTUP_INIT_VERSION` | Configure the version of rustup-init to install. It can be a specific version or a wildcard like `1.*`. It defaults to the latest `1.*` version.                                                                                                                             |
 | `$BP_RUSTUP_INIT_LIBC`    | Configure the libc implementation used by the installed toolchain. Available options: `gnu` or `musl`. Defaults to `gnu` for compatiblity.                                                                                                                                   |
```

Comment:
To have the best out of the box experience, we could also set `BP_RUSTUP_INIT_LIBC` to use `musl` when targeting the Tiny Stack.

Reply:
Could be either a new issue or done in this PR if the suggestion sounds good.

Reply:
I don't believe you want that. That is what caused the initial failure here -> https://github.com/paketo-community/rustup/issues/42#issue-1135923563

Reply:
But why would you ever set this then if it always results in that error?
Because logically for me it would make sense to use the musl rustup binaries for compiling a binary in the musl format with cargo.

Reply:
If it always fail I don't see why there would be the option to set this, maybe you could give an example when you would use this?

Reply:
When I added that option, I believe I had intended it to be used with Tiny but that doesn't seem to work. At this point, I can only guess, as I haven't tried this, but you might need that option if you were using a different stack. Perhaps one based off of an Alpine linux image, or another image that relies more on statically compiled binaries. I'll submit a second PR with a note to try and clarify that this option is a.) not used for Paketo buildpacks stacks and b.) add some notes specific for Tiny compatibility.